src/link/core.tsx

```diff
@@ -23,7 +23,7 @@ function LinkCore({
 }) {
   if (Platform.OS === 'web') {
     return (
-      <NextLink {...props} href={href} as={as} passHref>
+      <NextLink {...props} href={href} as={as} passHref legacyBehavior>
```

Comment:
Instead of legacyBehavior, I would prefer v1. It will be easier to understand which legacy version is being referenced.

Reply:
What do you mean? This prop just ensures that we’re using the RN element as the `a` tag.

Reply:
@nandorojo I was unable to deduce what the prop **legacyBehavior** does. Hence thought maybe we can have some better prop name

Reply:
This is a Next.js 13 prop that isn’t exposed to Solito users. https://nextjs.org/blog/next-13#breaking-changes